action/menu.js

```diff
@@ -0,0 +1,37 @@
+const installedScripts = [
+  {name: 'reblog_timestamps', title: 'Reblog Timestamps'},
+  {name: 'all_features', title: 'All Features'},
+];
+
+const scriptsList = document.querySelector('#scripts ul');
+const writeEnabled = () => {
+  const enabledScripts = installedScripts.map(
+    script => document.getElementById(script.name).checked ? script.name : undefined,
+  ).filter(x => x !== undefined);
+
+  browser.storage.local.set({
+    enabledScripts,
+  }).catch(console.error);
+};
+
+browser.storage.local.get('enabledScripts').then(({enabledScripts}) => {
+  enabledScripts = enabledScripts || [];
```

Comment:
we also need to default here—is there a good way to share the defaults between the popup and the content script?

Reply:
i was thinking of having a `gallery.json` file that stores all script metadata, might as well specify there if it's enabled by default or not

Reply:
the popup and the content script now technically default to nothing, but added a background script to set `enabledScripts`, so both the scripts are enabled upon installation and it's still possible to disable them both